app/src/main/res/layout/continue_interaction_item.xml

```diff
@@ -51,7 +52,7 @@
       app:layout_constraintTop_toTopOf="parent" />
 
     <org.oppia.android.app.customview.ContinueButtonView
-      android:id="@+id/animation_continue_button"
+      android:id="@+id/continue_button"
```

Comment:
Can this be more specific? View IDs are global in the app.

Reply:
done